sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -65,17 +77,22 @@
       printf("\n"); \
       printf(message); \
       printf("\n"); \
-      return 1; \
+      EXIT(1); \
```

Comment:
Again, I'd love to get rid of sample macros be they complicate customer understandability.

Reply:
Based on our offline conversation, I'll resurrect my PR to remove macros from samples.

Reply:
Agree that some Macros make things complicated.
But some times, it is super simple macros to avoid code duplication.

Also, macros are super idiomatic to C developers